process/slack_new_member_welcome.md

```diff
@@ -0,0 +1,42 @@
+<H1><img align="left" height=100 src="images/Girlgeek_w_workmark.png"><br> 
+  <em>Hello!</em></H1>
+
+&nbsp;  
+
+You're welcome to the Slack Platform of theGlobal PyLadies Community. 
+
+Kindly do well to introduce yourself in #introductions with the following information:
+
+
+
+- Your name
+- The PyLadies chapter you organize or belong to (_if you don't have a local one just say **"PyLadies"!**_)
+- Your pronouns (**e.g.** she/hers, they/them)
+- Tell us about your involvement with the Python Programming Language!
+
+
+
+While you can make posts anywhere on this platform, you are cautioned to put them in the right channel. We have many available channels, such as those listed below. However,  it may help you engage more effectively by finding the _appropriate_ channel to participate in! 
+
+Some great places to start: 
+
+\#introductions
+
+\#general
+
+\#conferences
+
+\#pyladies-support
+
+\#africa
+
+\#help-me
+
+
+
+
+If you are a PyLadies Organizer kindly do well to ask to be added to the private **organizer's** channel.
```

Comment:
"kindly do well to ask" - this is unusual English. I think it might be more common to say "please ask".

Reply:
Very minor detail: I think it's "organizers'" rather than "organizer's" (difference is many organizers vs. one organizer"